tests/package.json

```diff
@@ -12,6 +12,7 @@
     "@react-native-community/cli": "15.1.3",
     "@react-native-community/cli-platform-android": "15.1.3",
     "@react-native-community/cli-platform-ios": "15.1.3",
+    "@react-native-firebase/ai": "23.2.1",
```

Comment:
of course! the AI local test feature was the one I didn't test after porting manual tests into e2e app. No way it worked without this, sorry